src/main/scala/com/mozilla/telemetry/pings/MainPing.scala

```diff
@@ -107,6 +133,22 @@ case class MainPing(application: Application,
     dynamicProcessEvents.filter(_.category == "normandy")
   }
 
+  override def sessionSplitEvents: Seq[Event] = {
+    val timestamp: Int = meta.`payload.info` \ "sessionLength" match {
```

Comment:
Don't feel like you need to do this right now, but this would be better as a `sessionLength` getter in `Meta`